docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -418,26 +418,40 @@
 
           <para>The <emphasis role="bold">Set Banner</emphasis> link allows
           you to set a custom banner message at the top of your browser window
-          when you open ECL Watch. You can use this feature to send messages
-          to users.</para>
+          when you open ECL Watch. You check the <emphasis
```

Comment:
Check the Enable box to enable a banner. You can the banner to deliver a short message about the environment to users. Customize the appearance of the message banner with the other controls. Banner settings persist until the ESP Server restarts.